src/readthedocs_assistant/commands/migrate_config.py

```diff
@@ -93,7 +143,21 @@ async def fork_and_update(
         compare_url["html_url"],
     )
 
-    # TODO: Create pull request with message
+    accept = input("Continue? (yes/[no]) ") if interactive else "yes"
+    if accept == "yes":
+        pull_request = await gh.post(
+            f"/repos/{forked_repo['full_name']}/pulls",
```

Comment:
Instead of creating the PR shouldn't we expose the link to the user? I understood that we can't create PR.

I assume that this case worked because you created a PR on a repository under our own GH user.

Reply:
Yep, creating a repository that stays inside the fork works. I just checked that the base branch can't be changed...

I will move the message to the commit body to try to make this work.

Reply:
I moved the summary to the commit message and now if the human opens the pull request, the body is autofilled. The only problem is that the "howdy" intro is not included. But it felt lame to include that in the commit message... We'll have to do a bit of copy paste I assume.